tools.go

```diff
@@ -0,0 +1,21 @@
+// Copyright © 2023 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build tools
+
+package main
+
+import (
+_ "github.com/golangci/golangci-lint/cmd/golangci-lint"
```

Comment:
Since we're adding `tools.go` we should add the other tools used in this connector:

```suggestion
	_ "github.com/conduitio/conduit-connector-sdk/cmd/paramgen"
	_ "github.com/golangci/golangci-lint/cmd/golangci-lint"
	_ "go.uber.org/mock/mockgen"
```

And remove the `Makefile` variables `MOCKGEN_VERSION`, `PARAMGEN_VERSION` and `GOLANG_CI_LINT_VER`, as well as all targets that use them (`install-mockgen`, `install-paramgen` and `install-golangci-lint`.